src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,59 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id)
+                .orElseThrow(() -> new NoSuchElementException("There is no car by id.. "
```

Comment:
```suggestion
                .orElseThrow(() -> new NoSuchElementException("There is no car by id: "
```
то дві крапки просто для прикладу були)

Reply:
))